### ISSUE TITLE
nothing happens

### ISSUE THREAD

**Issue by jackfoxy** [sergey-tihon/vscode-openxml-explorer#6]
I believe I have successfully saved a large complex file in openxml format.
Nothing happens when I open it in VS Code.
OPENXML EXPLORER is visible but totally empty.
Any tips on debugging this?

**Comment by sergey-tihon** [comment]
Please update to v0.0.5. It contains a fix for server start + log channel if case of errors.
If it still does not work please tell me what is in the openxml log channel

**Comment by jackfoxy** [comment]
My problem is with LIbre Office. It says it is saving in open office xml format, but it does not

**Comment by sergey-tihon** [comment]
Just tested on files from Libre Office 6 and it works.
Can you share your file?

**Comment by jackfoxy** [comment]
https://github.com/arbrandt/OPGEE/blob/master/methane_model/Methane_Data_Gathering.xlsx is the file I attempted to save in office open xml from Libre Office 7.0.3.1 x64.

Maybe it works better in Libre Office 6

**Comment by jackfoxy** [comment]
I must be missing something. I just tried opening the test file excel-libre6.xlsx and I have the same problem. 
I see it is also in binary format, so I assume that is not the issue.

**Comment by sergey-tihon** [comment]
the file is binary because it is `zip` archive with folders, xml files and binary files. you always can change extension to `zip` and unarchive it manually.

I've tried to open your file and it work on my machine =)
<img width="479" alt="image" src="https://user-images.githubusercontent.com/1197905/114279314-3f8b4200-9a34-11eb-8d12-bc210d479769.png">

Do you extension on Windows? if yes update to `v0.0.6` we just released fix for windows #9

**Comment by jackfoxy** [comment]
v0.0.6
```

ExecError: Error: Command failed: dotnet Server.dll
Unhandled exception. System.IO.IOException: Failed to bind to address http://0.0.0.0:20489: address already in use.
 ---> Microsoft.AspNetCore.Connections.AddressInUseException: Only one usage of each socket address (protocol/network address/port) is normally permitted.
 ---> System.Net.Sockets.SocketException (10048): Only one usage of each socket address (protocol/network address/port) is normally permitted.
   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, String callerName)
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.Bind(EndPoint localEP)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.<Bind>g__BindSocket|13_0(<>c__DisplayClass13_0& )
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.<Bind>g__BindSocket|13_0(<>c__DisplayClass13_0& )
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportFactory.BindAsync(EndPoint endpoint, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.TransportManager.BindAsync(EndPoint endPoint, ConnectionDelegate connectionDelegate, EndpointConfig endpointConfig)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.<>c__DisplayClass29_0`1.<<StartAsync>g__OnBind|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.BindAsync(AddressBindContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.AddressesStrategy.BindAsync(AddressBindContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindAsync(IEnumerable`1 listenOptions, AddressBindContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.BindAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.WebHost.StartAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.RunAsync(IWebHost host, CancellationToken token, String startupMessage)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.RunAsync(IWebHost host, CancellationToken token, String startupMessage)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.RunAsync(IWebHost host, CancellationToken token)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.Run(IWebHost host)
   at Server.main(String[] _arg1) in /Users/sergey/github/vscode-openxml-explorer/src/Server/Server.fs:line 16
```

**Comment by sergey-tihon** [comment]
I've release v0.0.7 that dynamically choose port for api server and stop it on extension deactivation.
As a backup solution I added "Restart server" command, so you can restart it if something went wrong.

<img width="653" alt="image" src="https://user-images.githubusercontent.com/1197905/114512058-f6580f80-9c38-11eb-9c8a-a32ffd6889fb.png">

**Comment by jackfoxy** [comment]
I've loaded v0.0.7, and I am still feeling stupid because I never come close to seeing the kinds of screen images you have been posting.

I've tried opening the file directly.
I've tried opening the folder containing the file.
I've tried unpacking the compressed file to a folder and opening that folder.
I've tried opening the xml files within that folder.

I never see anything in the OpenXML Explorer. Is there some supporting thing I have not installed?

Here is my attempt to open the test xlsx:

![image](https://user-images.githubusercontent.com/1824476/114596892-0118a180-9c45-11eb-86a2-399474c57b53.png)
